code/src/components/StartPage.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react'
+import styled from 'styled-components';
+import { useDispatch } from 'react-redux'
+import Lottie from 'lottie-react';
+import { game, getGameStarted } from '../reducers/game'
+import { StyledButton } from './GlobalStyles';
+import animationData from '../lotties/rabbit.json';
+
+const StartWrapper = styled.div`
+display: flex;
+flex-direction: column;
+justify-content: space-evenly;
+height: 100vh;
```

Comment:
If you made the height of the StartWrapper less than 100vh (even just 95vh or 90vh), would it still overlap with the footer?